src/main/java/com/google/sps/servlets/FetchAnswersServlet.java

```diff
@@ -35,15 +38,97 @@
 /** 
  * Retrieves the answers to be displayed on the page.
  */
-@WebServlet("/answers")
+@WebServlet("/fetch-answers")
 public class FetchAnswersServlet extends HttpServlet {
```

Comment:
Why do you need a separate servlet for fetching versus posting answers?

Reply:
yeah, omar mentioned it to me in another PR and I wrote the TODO so that there is no merge conflict on my dependent branches right now

Reply:
Gotcha, thanks for clarifying!